content/docs/overview.md

```diff
@@ -1792,6 +1792,61 @@ f: Foo(T, N);
 #assert(size_of(f) == (N+N-2)*size_of(T));
 ```
 
+## Custom iterator functions
+
+Custom iterator functions can be used to iterate over anything you want.
+
+A custom iterator function should take a pointer to a struct keeps track of the state of the iteration.
```

Comment:
I'd rephrase this to:

```
It is customary for a custom iterator procedure to have a pointer to a value in order to keep track of the state of the iteration.
```

Reply:
Is "to a value" clear enough there? Like - `value` doesn't really convey anything about what it's about.

Reply:
Sure.